bluesky_queueserver/manager/start_manager.py

```diff
@@ -188,24 +188,32 @@ def run(self):
 
 def start_manager():
     parser = argparse.ArgumentParser(
-        description="Start a RE Manager", epilog=f"blueksy-queueserver version {__version__}"
+        description="Start Run Engine (RE) Manager", epilog=f"blueksy-queueserver version {__version__}"
     )
     parser.add_argument(
-        "--zmq_addr",
+        "--zmq-addr",
         dest="zmq_addr",
         type=str,
         default="tcp://*:5555",
         help="The address of ZMQ server.",
     )
-    parser.add_argument(
-        "--profile_collection",
-        "-p",
-        dest="profile_collection_path",
+
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
+        "--startup-dir",
+        dest="startup_dir",
+        type=str,
+        help="Path to directory that contains startup files (profile collection).",
+    )
+    group.add_argument(
+        "--startup-profile",
+        dest="profile_name",
         type=str,
-        help="Path to directory that contains profile collection.",
+        help="The name of IPython profile used to find the location of startup files (profile collection).",
```

Comment:
Also here, `(profile collection)` makes it ambiguous whether `profile_collection` or just `collection` are expected. How about `(example: collection)`.

Reply:
I added example to the help string.